draft-zsf.md

```diff
@@ -0,0 +1,176 @@
+```
+ZIP: 
```

Comment:
TODO ZIP Editors: Assign a ZIP number so that this ZIP can be referenced in PR #706.